browser/src/layer/marker/Cursor.ts

```diff
@@ -16,8 +16,7 @@ class Cursor {
 	headerName: string;
 	headerTimeout: number = 3000;
 
-	private position: cool.Point;
-	private size: cool.Point;
+	private rectangle: cool.SimpleRectangle;
```

Comment:
Makes sense, using a "point" type for the size was quite confusing, the new "rectangle" representation improves readability.